CordovaLib/CordovaLib.docc/upgrading-8.md

```diff
@@ -125,18 +125,17 @@ You can still access the `scrollView` property of the web view by dynamically in
 
 ```objc
 // Old code
-UIScrollView *scroller = self.webView.scrollView;
+UIScrollView *scrollView = self.webView.scrollView;
 ```
 
 ```objc
 // New code (Objective-C)
 #import <objc/message.h>
 
-UIScrollView *scroller;
-SEL scrollViewSelector = NSSelectorFromString(@"scrollView");
+UIScrollView *scrollView;
 
-if ([self.webView respondsToSelector:scrollViewSelector]) {
-    scroller = ((id (*)(id, SEL))objc_msgSend)(self.webView, scrollViewSelector);
+if ([self.webView respondsToSelector:@selector(scrollView)]) {
+    UIScrollView *scrollView = [self.webView performSelector:@selector(scrollView)];
```

Comment:
Right now this would be redeclaring `scrollView` rather than assigning it.

```suggestion
    scrollView = [self.webView performSelector:@selector(scrollView)];
```

Reply:
You are right, I removed the variable declaration before the `if` because a `nil` scrollView will not help, when there is no scrollView.